client/packages/programs/src/JsonForms/common/components/Array.tsx

```diff
@@ -169,12 +168,12 @@ const EnumArrayComponent: FC<EnumArrayControlCustomProps> = ({
                 borderRadius: '8px 8px 0px 0px',
               },
               '& .MuiChip-root': {
-                backgroundColor: theme.palette.secondary.light,
+                backgroundColor: 'secondary.light',
                 height: 'inherit',
-                color: theme.typography.login.color,
+                color: theme => theme.typography.login.color,
```

Comment:
Not sure I understand this change. What is different about it?
(And why don't the color ones above need to be like this?)

Reply:
the sx processing will convert strings to palette values for you, but only for known palette entries. The `login` is a custom one which we've added as an extension to the palette, so we need to call that directly from the theme. Using sx allows you to pass in a method to a color prop, which provides the theme object.

The values are really the same as before - but using the functionality provided by sx which means we don't require fetching the theme manually using the `useAppTheme` hook.